web/src/LogPane.tsx

```diff
@@ -110,6 +112,17 @@ class LogPane extends Component<LogPaneProps, LogPaneState> {
       )
     }
 
+    let resourceInfo: React.ReactElement
+
+    let endpoints = ""
+    if (this.props.endpoints) {
+      endpoints = this.props.endpoints.join(", ")
```

Comment:
it would be good to make each of these into links, instead of just text

Reply:
Agreed!